docs/user/masternodes/setup-testnet.rst

```diff
@@ -642,38 +642,34 @@ and communication relating to the base blockchain. Download Dash Core as
 follows::
 
   cd /tmp
-  wget https://github.com/dashpay/dash/releases/download/v19.2.0/dashcore-19.2.0-$(uname -m)-linux-gnu.tar.gz
+  wget https://github.com/dashpay/dash/releases/download/v19.3.0/dashcore-19.3.0-$(uname -m)-linux-gnu.tar.gz
 
 Verify the authenticity of your download by checking its detached
 signature against the public key published by the Dash Core development
-team. All releases of Dash are signed using GPG with one of the
-following keys:
+team. All releases of Dash are signed using GPG with the following key:
 
-- Alexander Block (codablock) with the key ``63A9 6B40 6102 E091``,
-  `verifiable here on Keybase <https://keybase.io/codablock>`__
 - Pasta (pasta) with the key ``5252 7BED ABE8 7984``, `verifiable here
   on Keybase <https://keybase.io/pasta>`__
 
 ::
 
-  curl https://keybase.io/codablock/pgp_keys.asc | gpg --import
   curl https://keybase.io/pasta/pgp_keys.asc | gpg --import
-  wget https://github.com/dashpay/dash/releases/download/v19.2.0/dashcore-19.2.0-$(uname -m)-linux-gnu.tar.gz.asc
-  gpg --verify dashcore-19.2.0-$(uname -m)-linux-gnu.tar.gz.asc
+  wget https://github.com/dashpay/dash/releases/download/v19.3.0/dashcore-19.3.0-$(uname -m)-linux-gnu.tar.gz.asc
+  gpg --verify dashcore-19.3.0-$(uname -m)-linux-gnu.tar.gz.asc
 
 Extract the compressed archive and copy the necessary files to the
 directory::
 
-  tar xfv dashcore-19.2.0-$(uname -m)-linux-gnu.tar.gz
-  sudo install -t /usr/local/bin dashcore-19.2.0/bin/*
+  tar xfv dashcore-19.3.0-$(uname -m)-linux-gnu.tar.gz
+  sudo install -t /usr/local/bin dashcore-19.3.0/bin/*
 
 Create a working directory for Dash Core::
 
   mkdir ~/.dashcore
 
 Configure Dash Core::
 
-  cat<<EOF>~/.dashcore/dash.conf
+  cat << EOF | tee ~/.dashcore/dash.conf
```

Comment:
> Is the intention to also echo the contents of the here-doc? Not sure why we are diverging from the canonical way of doing it.
> 
> ```
> cat > filename << "EOF"
> stuff
> EOF
> ```

The reason we did it like this was to use a consistent pattern with later `cat > filename` style commands, which require writing systemd files as the root user, which is achieved by running `sudo tee`. Happy for suggestions about more elegant ways to achieve this, maybe run the whole command with `sudo sh -c 'cat ...'`?